src/styles/globals.css

```
@@ -1155,3 +1155,725 @@
+  align-items: center;
+  gap: 12px;
+  flex-wrap: wrap;
+}
+.mdm-head__type {
+  display: inline-flex;
+  align-items: center;
+  gap: 6px;
+  height: 22px;
+  padding: 0 8px;
+  border-radius: 4px;
+  background: rgba(167, 139, 250, 0.10);
```

Stylelint (17.11.0)

[error] Expected "0.10" to be "10%" (alpha-value-notation)

(alpha-value-notation)